static/app/components/dateTime.tsx

```diff
@@ -74,14 +73,13 @@ export function DateTime({
       // UTC time.
       timeZone: timeZone ?? utc,
       seconds,
-      ...options,
```

Comment:
This was a mistake before. We were spreading the entire `user.options` object into the `getFormat` function.